tests/integration.rs

```diff
@@ -6728,6 +6748,46 @@ mod common_parallel {
 
         handle_child_output(r, &output);
     }
+
+    #[test]
+    #[cfg(target_arch = "x86_64")]
+    #[cfg(not(feature = "mshv"))]
+    fn test_tpm() {
+        let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+        let guest = Guest::new(Box::new(focal));
+
+        let (mut swtpm_command, swtpm_socket_path) = prepare_swtpm_daemon(&guest.tmp_dir);
+
+        let mut guest_cmd = GuestCommand::new(&guest);
+        guest_cmd
+            .args(["--cpus", "boot=1"])
+            .args(["--memory", "size=512M"])
+            .args(["--kernel", fw_path(FwType::RustHypervisorFirmware).as_str()])
+            .args(["--tpm", &format!("socket={swtpm_socket_path}")])
+            .capture_output()
+            .default_disks()
+            .default_net();
+
+        //start swtpm daemon
+        let mut swtpm_child = swtpm_command.spawn().unwrap();
+        thread::sleep(std::time::Duration::new(2, 0));
+        let mut child = guest_cmd.spawn().unwrap();
+        let r = std::panic::catch_unwind(|| {
+            guest.wait_vm_boot(None).unwrap();
+            assert_eq!(
+                guest.ssh_command("ls /dev/tpm0").unwrap().trim(),
```

Comment:
This command alone will not exercise the swtpm backend in CH. I think the sequence needs to be more sophisticated.

Reply:
I agree. As mentioned in a previous comment, I need a customized guest image to run additional checks.

I wanted to get a basic test merged without being blocked by custom guest image. So, I started with this one.